app/dashboard/views.py

```diff
@@ -1413,7 +1413,8 @@ def toolbox(request):
 
 
 def labs(request):
-    labs = LabsResearch.objects.all()
+    labs = LabsResearch.objects.filter(upcoming=False)
+    roadmap = LabsResearch.objects.all()
```

Comment:
Why are we querying for both of these?
Can we eliminate one or handle the `upcoming` check in template?

Reply:
Sure